test/parachain/src/cli.rs

```diff
@@ -95,22 +98,18 @@ where
 				config.network.listen_addresses = Vec::new();
 
 				// TODO
-				let mut polkadot_config =
-					polkadot_collator::Configuration::default_with_spec_and_base_path(
-						polkadot_service::ChainSpec::from_json_bytes(
-							&include_bytes!("../res/polkadot_chainspec.json")[..],
-						)?,
-						config.in_chain_config_dir("polkadot"),
-					);
+				let mut polkadot_config = parse_and_prepare::<NoCustom, NoCustom, _>(
```

Comment:
can the TODO above be removed? it's a bit weird to have a TODO where it's unspecified what should be done